### ISSUE TITLE
Disable button click while loading status is active

### ISSUE THREAD

**Issue by frabanca** [michaeldoye/mat-progress-buttons#36]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
```

### OS and Version?
Windows 10

### Versions
8.0.3

### Repro steps
1) Create a mat-progress-button
2) Build a back-end
3) Use the mat-progress-button as a submit form in your angular front-end 
4) Rush-click your button
5) Check you network tab. You will see one request for each click you have made, even if previous request were not completed.

Would be nice if buttons could be "disable" while in loading status.

### Desired functionality
Disable button click while loading status is active.

**Comment by michaeldoye** [comment]
This was actually fixed in a previous release ([8.0.3](https://github.com/michaeldoye/mat-progress-buttons/releases/tag/v8.0.3)), can you confirm which version you are using?

**Comment by frabanca** [comment]
Just tested and I can confirm that in the latest release this works like a charm! 
Thank you @michaeldoye for your work!
